### PR TITLE
Release Giant Swarm for Azure 8.6.0

### PR DESCRIPTION
**Pull request by omegas27**
Towards https://github.com/giantswarm/giantswarm/issues/7302

### CODE REVIEW

File: azure.yaml [hunk 1]
```diff
@@ -29,6 +29,21 @@
...
+    - name: cert-operator
+      version: 0.1.0
+    - name: chart-operator
+      version: 0.7.0
+    - name: cluster-operator
+      provider: azure
+      version: 0.20.0
+  date: 2019-12-31T10:00:00Z
+  version: 8.6.0
+- active: false
```
Comment (MarcelMue) [inline azure.yaml]: This should be true, right?

Comment (MarcelMue) [reply azure.yaml]: Wait no - I am dumb ignore me.

Comment (omegas27) [reply azure.yaml]: It is version 8.4.0. Do we want it active as 9.0.0 and 8.6.0 are active?

Comment (omegas27) [reply azure.yaml]: It happens :)

**Comment by omegas27** [comment]
@giantswarm/team-ludacris  should we close that one?

**Pull request closed without merging**